WHY_VIM.md

```diff
@@ -13,8 +18,14 @@
  - Amazing shortcuts and commands
 
  - You will always be learning new ways to speed up your workflow
+ 
+ - You will start to discover vim shortcuts built into all kinds of things
+ 
+ - It puts you into a good mindset for improving your workflow
```

Comment:
using vim trains you to _navigate_ in your editor/filesystem using your keyboard instead of a mouse. this means you can be _considerably_ faster at getting your work done.